src/Runner.jl

```diff
@@ -333,6 +333,14 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
             ])
             end
         end
+        if Sys.iswindows(p)
+            # Declare that we are Windows 10 (0x0A00)
+            # https://learn.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=msvc-170
+            append!(flags, [
+                "-DWINVER=0x0A00",
+                "-D_WIN32_WINNT=0x0A00",
+            ])
+        end
```

Comment:
I'm not very happy about duplication, it's redundant and error-prone.  Define a function (`windows_cflags`?) and use it in the two functions.